driver-kotlin-sync/src/test/kotlin/com/mongodb/kotlin/client/MongoIterableTest.kt

```diff
@@ -93,7 +93,7 @@ class MongoIterableTest {
         assertContentEquals(documents.subList(0, 2), iterable.use { it.take(2) }.toList())
 
         verify(delegate, times(1)).cursor()
-        verify(cursor, times(2)).hasNext()
+        verify(cursor, times(3)).hasNext()
```

Comment:
Due to a change in the kotlin internals - it now checks `hasNext` 3 times. But only calls `next` twice.